blockstore/apps/search/tests/test_views.py

```diff
@@ -0,0 +1,229 @@
+""" Tests for views. """
+
+from elasticsearch.exceptions import NotFoundError
+import pytest
+
+from ..core.tests.test_base import ViewsBaseTestCase
+from ..documents import BlockDocument
+from ..fieldsets import (
+    SummaryFieldSet,
+)
+
+
+class BlockDocumentsViewSetTestCase(ViewsBaseTestCase):
+    """ Tests for BlockDocumentsViewSet. """
+
+    document_class = BlockDocument
+
+    list_path = 'api:v1:index-blocks-list'
+    detail_path = 'api:v1:index-blocks-detail'
+
+    def test_list(self):
+
+        response = self.response(self.list_path)
+        assert response.data['results'] == []
```

Comment:
I believe we shouldn't use the `assert` keyword since it can be optimized out... https://stackoverflow.com/a/6361186

Reply:
Hmm. The pytest examples all use `assert`: https://docs.pytest.org/en/latest/. But I have updated all instances to `self.assertEqual()` for consistency.